zoom/tools/meetings.py

```diff
@@ -416,6 +427,42 @@ def update_meeting():
             )
         payload["type"] = meeting_type
 
+    # args of settings
+    settings = {}
+    if "AUDIO_RECORDING" in os.environ:
+        audio_recording = os.environ["AUDIO_RECORDING"]
+        settings["auto_recording"] = audio_recording
+    if "CONTACT_EMAIL" in os.environ:
+        contact_email = os.environ["CONTACT_EMAIL"]
+        settings["contact_email"] = contact_email
+    if "CONTACT_NAME" in os.environ:
+        contact_name = os.environ["CONTACT_NAME"]
+        settings["contact_name"] = contact_name
+    if "PRIVATE_MEETING" in os.environ:
+        private_meeting = str_to_bool(os.environ["PRIVATE_MEETING"])
+        settings["private_meeting"] = private_meeting
```

Comment:
nit: any reason not to do this?

```suggestion
    if "AUDIO_RECORDING" in os.environ:
        settings["auto_recording"] = os.environ["AUDIO_RECORDING"]
    if "CONTACT_EMAIL" in os.environ:
        settings["contact_email"] = os.environ["CONTACT_EMAIL"]
    if "CONTACT_NAME" in os.environ:
        settings["contact_name"] = os.environ["CONTACT_NAME"]
    if "PRIVATE_MEETING" in os.environ:
        settings["private_meeting"] = str_to_bool(os.environ["PRIVATE_MEETING"])
```

Also, is the param supposed to be called `auto_recording`, or `audio_recording`? I'm not sure if the inconsistency is in the name of the env var or of the parameter, but they should be changed to match.

Reply:
> nit: any reason not to do this?
> 
> Also, is the param supposed to be called `auto_recording`, or `audio_recording`? I'm not sure if the inconsistency is in the name of the env var or of the parameter, but they should be changed to match.

and yes, we should simplify these. it was there probably because some of the env needs condition checks.
Fixed as requested.